python/pyspark/testing/connectutils.py

```diff
@@ -28,7 +28,7 @@
     from pyspark.sql.connect.plan import LogicalPlan
     from pyspark.sql.connect.session import SparkSession
 
-    connect_jar = search_jar("connector/connect", "spark-connect-assembly-", "spark-connect")
+    connect_jar = search_jar("connector/connect/server", "spark-connect-assembly-", "spark-connect")
```

Comment:
Thanks. Looks like the trick is to find the server assembly jar which include the proto generated code, the server and necessary spark code.

Reply:
Yup.